Flow.Launcher/Resources/CustomControlTemplate.xaml

```diff
@@ -2802,7 +2802,6 @@
                         </Trigger>
                         <Trigger Property="IsEnabled" Value="false">
                             <Setter TargetName="HeaderContentPresenter" Property="Foreground" Value="{DynamicResource TextControlHeaderForegroundDisabled}" />
-                            <Setter Property="Background" Value="{DynamicResource CustomNumberBoxBGDisabled}" />
```

Comment:
I had worked on this key, but now I see it’s not used anywhere. Since the intended color is being displayed correctly, I’ve removed this part.